schema.production.kf

```diff
@@ -461,6 +622,70 @@ action share_credential_through_dag (
     INSERT INTO shared_credentials (original_id, duplicate_id) VALUES ($original_credential_id, $id);
 }
 
+// TODO: creds2: I can't find who's using this. Could very well be nobody.
+action share_credential_through_dag2 (
+  $id
+, $human_id
+, $issuer
+, $encryption_public_key
+, $content
+, $public_notes
+, $public_notes_signature
+, $broader_signature
+, $original_credential_id
+, $dag_owner
+, $dag_grantee
+, $dag_locked_until
+, $dag_signature
+) public {
+    SELECT CASE WHEN NOT EXISTS (
+        SELECT 1 from credentials2
+            WHERE id = $original_credential_id
+            AND human_id = $human_id
+    ) THEN
+        error('The original credential does not belong to the human')
+    END;
+
+    $owner_verified = idos.verify_owner($dag_owner, $dag_grantee, $id, $dag_locked_until, $dag_signature);
+    SELECT CASE WHEN $owner_verified != 1 THEN
+        error('the signature is invalid')
```

Comment:
As now we have a loooot of signatures i would name the error as `the dag_signature is invalid`

Reply:
Great point. :ok_hand: